src/viewer.js

```diff
@@ -295,6 +295,8 @@ $.Viewer = function( options ) {
     this.container.insertBefore( this.canvas, this.container.firstChild );
     this.element.appendChild( this.container );
 
+
```

Comment:
There are a few places like this where you have added extra blank lines. It would be cleaner to remove them.